scripts/run_with_manager.py

```diff
@@ -0,0 +1,41 @@
+
+import asyncio
+import logging
+from astropy.time import Time, TimeDelta
+import astropy.units as u
+from process_manager import TaskType, ProcessManager, Scheduler
+from process_manager import SchedulerConfig, CollectorConfig, SelectorConfig
+from api.observatory.gemini import GeminiProperties
+from common.minimodel import ALL_SITES, Semester, SemesterHalf, ProgramTypes, ObservationClass
+
+logging.basicConfig(level=logging.INFO)
```

Comment:
We should probably still have this in:
```
if __name__ == '__main__':
```
since it's Python convention to make all executables check for that before running.